packages/calcite-components/src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -72,8 +92,43 @@ export class DatePickerMonth {
    */
   @Prop() localeData: DateLocaleData;
 
-  /** The range of dates currently being hovered. */
-  @Prop() hoverRange: HoverRange;
+  /** Specifies the latest allowed date (`"yyyy-mm-dd"`). */
+  @Prop() max: Date;
+
+  /**
+   * Made into a prop for testing purposes only
+   *
+   * @internal
+   */
+  // eslint-disable-next-line @stencil-community/strict-mutable -- updated by t9n module
```

Comment:
Mutable and the disable line shouldn't be necessary since this is not implementing `T9nComponent`.